src/spryker/docker/image/console/root/lib/task/install.sh.twig

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+function task_install()
+{
+    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -s database-migrate"
+    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -s demodata"
+    task "overlay:apply"
+    task "assets:dump"
+}
```

Comment:
same here https://github.com/inviqa/harness-base-php/blob/0.2.x/src/_base/docker/image/console/root/lib/task/install.sh.twig#L7

Reply:
wilco

Reply:
you could make task_* scripts for spryker init/build/install to encapuslate the logic so it can be a single step

Reply:
good idea, will do